### PR TITLE
Update copyright statements

### PR DESCRIPTION
**Pull request by franzliedke**
This updates the copyright statements, as the project has now been transferred to the Flarum Foundation (Stichting Flarum).

This is my proposal. Let's discuss the header we want here - code changes can then be enforced with StyleCI.

This needs approval from @tobyzerner, too.

### CODE REVIEW

**Comment by franzliedke** [comment]
Haha, I knew this would happen. :roll_eyes: 

Please review the commits one-by-one - StyleCI already reverted the change to the code file, so the total diff does not show my complete proposal.

**Comment by tobyzerner** [comment]
Looks good to me 👍

**Comment by franzliedke** [comment]
Open questions from my side:
- "Stichting Flarum" or "Flarum Foundation"?
- Does the "based on code" make sense in the `LICENSE` file?

**Comment by jordanjay29** [comment]
> Open questions from my side:
> 
>     * "Stichting Flarum" or "Flarum Foundation"?

I think we definitely need the former for legal clarity, but the latter is more clear for most non-Dutch people. Perhaps `Stichting Flarum (Flarum Foundation)`

>     * Does the "based on code" make sense in the `LICENSE` file?

No, if I didn't know the context, I would have no idea what this meant. It might be better to append the years of those copyrights such as: 
```
Copyright (c) 2019 Stichting Flarum 
Copyright (c) 2014-2019 Toby Zerner (toby.zerner@gmail.com)
```

**Comment by datitisev** [comment]
Not sure if this matters, but it'd probably make more sense to not include the "based on code" in the files themselves. The license comment does reference the LICENSE file, so not sure if that's necessary or if we just want it there.

Also yeah, I agree that copyright years might be better.

**Comment by luceos** [comment]
I'm with @jordanjay29 on this.

**Comment by franzliedke** [comment]
Okay, I added the years to the LICENSE file, and removed copyright information from the file headers. @tobyzerner mentioned this possibility in chat at some time.

Is that fine for everyone?

File: LICENSE [hunk 1]
```diff
@@ -1,6 +1,7 @@
 The MIT License (MIT)
 
-Copyright (c) Toby Zerner
+Copyright (c) 2019 Stichting Flarum
```
Comment (jordanjay29) [inline LICENSE]:
```suggestion
Copyright (c) 2019 Stichting Flarum (Flarum Foundation)
```

**Comment by tobyzerner** [comment]
👍

**Comment by franzliedke** [comment]
Okay, I configured the proposed code file header in StyleCI, but it didn't apply it. :thinking:

**Comment by franzliedke** [comment]
Ah, apparently StyleCI needed a commit to trigger the change. So I pushed [the LICENSE file change](https://github.com/flarum/core/commit/19188e3eda6b34106c6f771e4ff15f219bc57270) to master now, and it followed up with [the code file changes](https://github.com/flarum/core/commit/d492579638fb52dafbfe65f1f36a95eb6047f7f3).

**Pull request closed without merging**